edx_solutions_api_integration/users/views.py

```diff
@@ -315,7 +316,17 @@ def get_queryset(self):
             org_ids = map(int, org_ids.split(','))
             queryset = queryset.filter(organizations__id__in=org_ids).distinct()
 
-        if match == 'partial':
+        if match == 'extendedpartial':
```

Comment:
Put a comment for what is `extendedpartial` and may be rename it to `extended_partial`

Reply:
also please update the doctstring for this method

Reply:
Ok